drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/actualcommands/AbstractSetHeaderCommand.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.scenariosimulation.client.commands.actualcommands;
+
+import java.util.Optional;
+
+import org.drools.workbench.screens.scenariosimulation.client.commands.ScenarioSimulationContext;
+import org.drools.workbench.screens.scenariosimulation.client.metadata.ScenarioHeaderMetaData;
+import org.drools.workbench.screens.scenariosimulation.client.resources.i18n.ScenarioSimulationEditorConstants;
+import org.drools.workbench.screens.scenariosimulation.client.widgets.ScenarioGridColumn;
+import org.drools.workbench.screens.scenariosimulation.model.FactIdentifier;
+import org.drools.workbench.screens.scenariosimulation.model.ScenarioSimulationModel;
+
+import static org.drools.workbench.screens.scenariosimulation.model.FactMapping.getPropertyPlaceHolder;
+
+/**
+ * <b>Abstract</b> <code>Command</code> class to provide common methods used by <code>SetInstanceHeaderCommand</code> and <code>SetPropertyHeaderCommand</code> implementations
+ */
+public abstract class AbstractSetHeaderCommand extends AbstractScenarioSimulationCommand {
+
+    public AbstractSetHeaderCommand() {
+        super(true);
+    }
+
+    protected abstract void conditionalExecute(ScenarioSimulationContext context, ScenarioGridColumn selectedColumn);
```

Comment:
Can you please rename it to `executeIfSelectedColumn`? I think it is more understandable

Reply:
Done